### PR TITLE
Revert Fluentd from 1.12.3 to 1.11.2 for memory / cpu regression.

### PR DESCRIPTION
**Pull request by qingling128**
Fixes https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/447#issuecomment-847187748

### CODE REVIEW

File: fluent-plugin-google-cloud.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'fluentd', '1.12.3'
+  gem.add_runtime_dependency 'fluentd', '1.11.2'
```
Comment (igorpeshansky) [inline fluent-plugin-google-cloud.gemspec:22]: We also need to update https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/config/software/fluentd.rb

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

File: fluent-plugin-google-cloud.gemspec (hunk 1 above)
Comment (qingling128) [reply fluent-plugin-google-cloud.gemspec:24]: Done in https://github.com/GoogleCloudPlatform/google-fluentd/pull/340. Also added a note for the two locations to update.

**Comment by qingling128** [comment]
Filed https://github.com/fluent/fluentd/issues/3389 in the Fluentd upstream repo.

**Pull request merged**